src/main/java/com/targomo/client/api/geo/Location.java

```diff
@@ -32,6 +32,18 @@ public interface Location {
      */
     public void setId(final String id);
 
+    /**
+     * Get the H3 address that represents a coordinate.
+     * @return The H3 Address
+     */
+    public String getH3Address();
+
+    /**
+     * Sets the H3 address that represents a coordinate.
+     * @param h3Address The H3 Address
+     */
+    public void setH3Address(final String h3Address);
```

Comment:
do you really need to have add those methods in the interface? I would only add them to `DefaultSourceAddress `

Reply:
Yes because they allow me to use the `DefaultSourceAddress` for the methods that also use the default coordinates and geometries since they are all locations

Reply:
Actually let me check

Reply:
in your code each time you use `setH3Address` or `getH3Address` it is with a `DefaultSourceAddress`
so you don't need it in the interface for me